<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Recorder_json</class>
 <widget class="QMainWindow" name="Recorder_json">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Recorder_json</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="userText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>341</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="jsonText">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>30</y>
      <width>421</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addJson_File">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>addJson_File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addJson_Key">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>addKey</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearJson_File">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>460</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveJson_File">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>111</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveJson_File</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
